MIGRATION.md

```diff
@@ -0,0 +1,11 @@
+# Migration
+
+## 1.2.1 → 2.0.0
+
+No post-upgrade action is required if you are using a linter without a custom
+configuration file.
+
+If you are using a custom configuration file, then you should check the
```

Comment:
I'm not sure, but that line seems weird, maybe it's just Present Simple here, not Continuous?

And I think there's no need in "the" articles here either.